### ISSUE TITLE
Too small size of back button

### ISSUE THREAD

**Issue by uxes** [ubports/ubuntu-touch#2122]
### Description of the feature
As a grown man with big fingers, i would love to have back button to have bigger area to be clicked on, ideal size could be that one that is in cinny application

### Illustrations

too small size of element
![image](https://user-images.githubusercontent.com/1339083/228941939-136d2ced-8aa7-4ff7-bb8c-2d4417c2a4d8.png)

the ideal size
![image](https://user-images.githubusercontent.com/1339083/228942229-9f94644c-e79e-4db0-ae04-944b78554006.png)

**Comment by lduboeuf** [comment]
I agree a bit small for me as well. but changing that is not as easy as it seems to be.

**Comment by r4vanan** [comment]
i am agree with you. same feeling for me too

**Comment by eMPee584** [comment]
> I agree a bit small for me as well. but changing that is not as easy as it seems to be.

Well where is the size defined? wouldn't QML allow overriding this with sort of a user stylesheet?

**Comment by lduboeuf** [comment]
> > I agree a bit small for me as well. but changing that is not as easy as it seems to be.
> 
> Well where is the size defined? wouldn't QML allow overriding this with sort of a user stylesheet?

Somewhere here https://gitlab.com/ubports/development/core/lomiri-ui-toolkit/-/tree/main ...

**Comment by lduboeuf** [comment]
> > > I agree a bit small for me as well. but changing that is not as easy as it seems to be.
> > 
> > 
> > Well where is the size defined? wouldn't QML allow overriding this with sort of a user stylesheet?
> 
> Somewhere here https://gitlab.com/ubports/development/core/lomiri-ui-toolkit/-/tree/main ...

more precisely: https://gitlab.com/ubports/development/core/lomiri-ui-toolkit/-/blob/main/src/imports/Components/Themes/Ambiance/1.3/PageHeaderStyle.qml#L95

But it is general to all actions we see in the header, so need special handling, test on small screen and big ones...


We could maybe better override the back Action here: https://gitlab.com/ubports/development/core/lomiri-ui-toolkit/-/blob/main/src/imports/Components/1.3/AdaptivePageLayout.qml#L717
